app/models/miq_request_workflow.rb

```diff
@@ -200,6 +200,10 @@ def validate(values)
           # TODO: do we need validation for boolean
         when :button
           # Ignore
+        when :array_integer
+          unless is_array?(value)
```

Comment:
@gmcculloug - Like you said - the naming was copying a pattern which coincides with the other validation methods e.g. `is_integer` etc. I know we wanted to make the data_type name `:array_integer` so we could validate that the data in the array were all integers, which this code is not doing.  So with that in mind I have two distinct questions.

Should we just rename the data_type to `:array` and let this code do what it currently does? https://github.com/syncrou/manageiq/blob/92fe9f127b7471ac529e27d329f49211dc3307fe/app/models/miq_request_workflow.rb#L208-L211

Or

Should we keep the type `:array_integer`, and have the `is_array?` (or the new name of the method) method handle all the validation required of that type. Being that the data is an array of integers?